src/mixins/filing-template-mixin.ts

```diff
@@ -1,20 +1,26 @@
 // Libraries
 import { Component, Vue } from 'vue-property-decorator'
 import { Action, Getter, State } from 'vuex-class'
+
 // Interfaces
 import {
   ActionBindingIF,
+  AlterationFilingIF,
+  BusinessSnapshotIF,
   CorrectionFilingIF,
   IncorporationFilingIF,
   OrgPersonIF,
   ShareClassIF,
   StateModelIF,
   NameTranslationIF,
-  NameTranslationDraftIF
+  NameTranslationDraftIF,
+  RoleIF
 } from '@/interfaces'
+
 import { StaffPaymentIF } from '@bcrs-shared-components/interfaces'
+
 // Constants
-import { ActionTypes, EntityTypes, FilingTypes } from '@/enums'
+import { ActionTypes, EntityTypes, FilingTypes, Roles } from '@/enums'
```

Comment:
Could you please rename Roles to RoleTypes (as I've done in another project)?

Reply:
Bit outside scope but sure..
Also Unused import, removed 👍

Reply:
You're right. This cleanup can be done later. Thanks either way :)

Reply:
Thanks! 🙏